{
  try {
    qt.clearPostTestEffects();
  }
 catch (  Exception e) {
    System.err.println("Exception: " + e.getMessage());
    e.printStackTrace();
    System.err.flush();
    fail("Unexpected exception in tearDown");
  }
}
